Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.9-slim-buster as base
+# Install poetry
+RUN pip install poetry
+WORKDIR /app
+# Expose the port that we want the host to listen to
+EXPOSE 5000
+# Copy across the project files
+COPY . /app
+# Install the project dependencies
+RUN poetry install
```

Comment:
Minor but worth noting, we could swap this to only install the prod-relevant dependencies with `poetry install --no-dev` so that our production image doesn't have any unnecessary components (like pytest) but we want to add a full poetry install into the development stage if we took that approach